tools/bin/hawqregister

```diff
@@ -297,8 +297,8 @@ def move_files_in_hdfs(databasename, tablename, files, firstsegno, tabledir, nor
     '''Move file(s) in src path into the folder correspoding to the target table'''
     if normal:
         segno = firstsegno
-        for file in files:
-            srcfile = file
+        for f in files:
+            srcfile = f
```

Comment:
Forget it since I was talking about the --force option implementation. It's out of scope of this PR.